app/assets/stylesheets/lifehack_style.scss

```diff
@@ -0,0 +1,160 @@
+.form-text {
+  float: left;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 15px;
+}
+
+.lifehack-details {
+  max-width: 300rem;
+  padding: rem-calc(50);
+}
+
+.panel {
+
+  h5 {
+    font-size: 30px;
+    text-align: justify;
+    padding: 23px;
+  }
+
+  h2 {
+    border-style: double;
+    color: $primary-color;
+    font-size: 30px;
+    padding:  40px 2.875rem 40px 2.875rem;
+    text-align: left;
+    }
+
+  h3 {
+    color: $black;
+    font-size: 30px;
+    padding:  9px 2.875rem 10px 2.875rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `9px 2.875rem 10px` instead of `9px 2.875rem 10px 2.875rem`<br>Colon after property should be followed by one space